.golangci.yml

```diff
@@ -43,7 +43,7 @@ issues:
   exclude-use-default: false
   # List of regexps of issue texts to exclude, empty list by default.
   exclude:
-    - Using the variable on range scope `(tc)|(rt)|(tt)|(test)|(testcase)|(testCase)` in function literal
+    - Using the variable on range scope .* in function literal
```

Comment:
Not a real fix, but scopelint should be replaced by exportloopref via https://github.com/kubernetes-sigs/cluster-api/pull/4649 anyway